### PR TITLE
man: fix info

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: man/oci-runtime-tool-generate.1.md [hunk 1]
```diff
@@ -337,24 +337,24 @@ This protects the containers image from modification. Read only containers may
 still need to write temporary data.  The best way to handle this is to mount
 tmpfs directories on /generate and /tmp.
 
-    # oci-runtime-tool generate --read-only --tmpfs /generate --tmpfs /tmp --tmpfs /run  --rootfs /var/lib/containers/fedora /bin/bash
+    $ oci-runtime-tool generate --read-only --tmpfs /generate --tmpfs /tmp --tmpfs /run  --rootfs /var/lib/containers/fedora /bin/bash
```
Comment (Mashimiao) [inline man/oci-runtime-tool-generate.1.md]: the options are too old, please update them

File: man/oci-runtime-tool-generate.1.md [hunk 2]
```diff
@@ -364,23 +364,23 @@ To mount a host directory as a container volume, specify the absolute path to
...
 
-    # oci-runtime-tool generate --bind /var/db:/data1  --selinux-label system_u:system_r:svirt_lxc_net_t:s0:c1,c2 --mount-label system_u:object_r:svirt_sandbox_file_t:s0:c1,c2 --rootfs /var/lib/containers/fedora --args bash
+    $ oci-runtime-tool generate --bind /var/db:/data1  --selinux-label system_u:system_r:svirt_lxc_net_t:s0:c1,c2 --mount-label system_u:object_r:svirt_sandbox_file_t:s0:c1,c2 --rootfs /var/lib/containers/fedora --args bash
 
 Not in the above example we used a type of svirt_lxc_net_t and an MCS Label of s0:c1,c2.  If you want to guarantee separation between containers, you need to make sure that each container gets launched with a different MCS Label pair.
 
 Also the underlying rootfs must be labeled with a matching label.  For the example above, you would execute a command like:
 
-    # chcon -R system_u:object_r:svirt_sandbox_file_t:s0:c1,c2  /var/lib/containers/fedora
+    $ chcon -R system_u:object_r:svirt_sandbox_file_t:s0:c1,c2  /var/lib/containers/fedora
```
Comment (Mashimiao) [inline man/oci-runtime-tool-generate.1.md]: chcon must be executed by root

Comment (q384566678) [inline man/oci-runtime-tool-generate.1.md]: updated

**Comment by Mashimiao** [comment]
LGTM

**Comment by liangchenye** [comment]
LGTM

**Pull request merged**